data/mods/Medieval_Stuff/tools.json

```diff
@@ -0,0 +1,543 @@
+[
+  {
+    "id": "arming_sword",
+    "type": "TOOL",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "arming sword",
+    "description": "This is a classic medieval sword, just the right size to use one-handed.",
+    "price": 100000,
+    "material": "steel",
+    "flags" : ["CHOP", "DURABLE_MELEE", "SHEATH_SWORD"],
+    "techniques": ["RAPID", "WBLOCK_1"],
+    "weight": 2721,
+    "volume": 10,
+    "bashing": 11,
+    "cutting": 37,
+    "to_hit": 2,
+    "max_charges": 0,
+    "initial_charges": 0,
+    "charges_per_use": 0,
+    "turns_per_charge": 0,
+    "ammo": "NULL",
+    "revert_to": "null",
+    "use_action": { "type": "knife" },
+    "category" : "weapons",
+    "qualities": [["CUT", 1], ["BUTCHER", 8]]
+  },
+  {
+    "id": "longsword_one",
+    "type": "TOOL",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "longsword (one-handed stance)",
+    "name_plural": "longswords (one-handed stance)",
+    "description": "This is a classic medieval sword, in between the lighter arming sword and later two-handed swords in size.  It can be used either one-handed, or two-handed.",
+    "price": 130000,
+    "material": "steel",
+    "flags" : ["CHOP", "DURABLE_MELEE", "SHEATH_SWORD"],
+    "techniques": ["RAPID", "WBLOCK_1"],
+    "weight": 2948,
+    "volume": 12,
+    "bashing": 13,
+    "cutting": 38,
+    "to_hit": 2,
+    "max_charges": 0,
+    "initial_charges": 0,
+    "charges_per_use": 0,
+    "turns_per_charge": 0,
+    "ammo": "NULL",
+    "revert_to": "null",
+    "use_action": [
+        {
+            "type": "knife"
+        },
+        {
+            "type": "auto_transform",
+            "msg": "You shift to a two-handed stance.",
```

Comment:
This would be better as a style.

Also, losing rapid strike by adding an extra hand doesn't make any amount of sense.